
We at least do not want the .csh file in /etc/profile.d - we probably need to
create an own one, but do not want to use the shipped one for bash.

  - Rene Rebe <rene@rocklinux.org>

--- ./programs/Xserver/Xprint/etc/profile.d/Imakefile.orig	2004-08-25 19:27:47.950043112 +0200
+++ ./programs/Xserver/Xprint/etc/profile.d/Imakefile	2004-08-25 19:27:55.620876968 +0200
@@ -10,5 +10,3 @@
 
 all::
 
-InstallNamedProg(xprint.csh,xprint.csh,$(ETCDIR)/profile.d)
-InstallNamedProg(xprint.sh,xprint.sh,$(ETCDIR)/profile.d)
